packages/ui/src/components/ContactName.tsx

```diff
@@ -139,7 +139,13 @@ export default function ContactName({
   }
 
   return (
-    <SizableText aria-label={formattedId.ariaLabel} {...rest}>
+    <SizableText
+      ellipsizeMode="tail"
+      numberOfLines={1}
+      maxWidth="75%"
```

Comment:
Will we always want this `maxWidth` here? If not, maybe we pass that in at the call site where needed

Reply:
I think it's probably ok, it'll just make sure that we keep a reasonable length anywhere that the parent has a set width, afaik